README.md

```diff
@@ -118,6 +118,27 @@ Then to verify the pipeline definition using the defined policies:
 
     make check-pipeline
 
+### Running policies against local [ec-cli] build
+
+Build a local version of `ec-cli` in your local ec-cli repo:
+
+    make build
+
+Create a `policy.yaml` file in your local `ec-cli` repo with something like:
+
+    ---
+    sources:
+      - policy:
+        - github.com/enterprise-contract/ec-policies//policy//lib
+        - github.com/enterprise-contract/ec-policies//policy//release
+      data:
+        - oci::quay.io/konflux-ci/tekton-catalog/data-acceptable-bundles:latest
+        - github.com/release-engineering/rhtap-ec-policy//data
+
+Run the locally built `ec-cli` command
```

Comment:
Could also do `go run github.com/enterprise-contract/ec-cli validate image ...` and skip building. We have a make target called `ec-version`, perhaps we can create a `ec` target to help running ec CLI a bit simpler

Reply:
This will use the version of ec-cli that is referenced [here](https://github.com/enterprise-contract/ec-policies/blob/540adf985ae1432fc16129a7bc5676222da9d1c3/go.mod#L6) which may not be the intention. Someone trying to use a locally built ec-cli is likely trying to test local changes to the ec-cli.